com.rawtherapee.RawTherapee.yaml

```diff
@@ -20,6 +20,8 @@ finish-args:
   # Access to temporary files
   # Share data between RawTherapee and external editors, especially GIMP
   - --filesystem=/tmp
+  # Allow access to the host's colord
+  - --system-talk-name=org.freedesktop.ColorManager
```

Comment:
I fail to see how it can be related to **RawTherapee**.

**Darktable** uses `colord`, so it makes sense for it to access `org.freedesktop.ColorManager`.
https://github.com/flathub/org.darktable.Darktable/blob/73e1c984212ff7aa723adb753e6c83860a37af1b/org.darktable.Darktable.json#L17-L18
https://github.com/flathub/org.darktable.Darktable/blob/73e1c984212ff7aa723adb753e6c83860a37af1b/org.darktable.Darktable.json#L111-L141
https://github.com/darktable-org/darktable/blob/release-3.0.0/src/CMakeLists.txt#L684-L708

**RawTherapee** doesn't, so what it the point of this PR?

First you have to implement it.
https://github.com/Beep6581/RawTherapee/issues/4335

It shouldn't be so hard, but someone have to do this and test it.

See also:
https://github.com/Beep6581/RawTherapee/pull/3009#issuecomment-162324644